services/frontend/src/pages/HeroDetailPage.tsx

```
@@ -0,0 +1,243 @@
+  )
+  const itemsSection = useSection(() => computedApi.getHeroItems(heroId, primaryPos))
+  const matchupsSection = useSection(() => computedApi.getHeroMatchups(heroId, 10))
+  const countersSection = useSection(() => computedApi.getHeroCounters(heroId, 10))
+  const synergiesSection = useSection(() => computedApi.getHeroSynergies(heroId, 10))
+
+  const heroName = statsSection.data?.hero_name ?? `Hero #${heroId}`
+
+  return (
+    <div>
+      <div className="detail-header">
+        <button className="back-btn" onClick={() => navigate('/')}>← Back</button>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)